connectors/salesforce/src/test/resources/connector/responses/successSenderDetails_xx.json

```diff
@@ -22,7 +22,7 @@
           {
             "type": "GENERAL",
             "title": "Konto",
-            "description": "Uber Inc"
+            "description": "Uber IT Account"
```

Comment:
It was easy to match with the test information in `jjeff` Salesforce dev account.